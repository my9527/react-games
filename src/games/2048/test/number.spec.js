import React from 'react';
import CmptNumber from '../components/number';
import renderer from 'react-test-renderer';


test('CmptNumber', () => {

    const n = renderer.create(
        <CmptNumber num={5}/>
    )

    let _n = n.toJSON();
    expect(_n).toMatchSnapshot();
})