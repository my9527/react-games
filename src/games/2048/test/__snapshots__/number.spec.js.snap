// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CmptNumber 1`] = `
<div
  className="cmpt cmpt-number"
  style={
    Object {
      "backgroundColor": undefined,
      "color": "white",
    }
  }
>
  5
</div>
`;
